src/app/chat-prompt-component/chat-prompt-component.component.html

```diff
@@ -7,7 +7,7 @@
       <textarea #messageInputArea class="textarea-message" rows="1" [(ngModel)]="messageInput" cdkTextareaAutosize
                 cdkAutosizeMinRows="1"
                 cdkAutosizeMaxRows="3" type="text" placeholder="Type a message..."
-                (keydown.enter)="$event.preventDefault();sendMessage.emit(messageInput); messageInput = '';"></textarea>
+                (keydown.enter)="$event.preventDefault();messageInput.trim() ? sendMessage.emit(messageInput) : null; messageInput = '';"></textarea>
```

Comment:
i would not trim here maybe people want to input whitespaces for whatever reason